.github/ISSUE_TEMPLATE/4-release.yml

```diff
@@ -0,0 +1,125 @@
+name: 'WooCommerce: release checklist'
+description: Create a new issue to track the next WooCommerce core release progress.
+title: "WooCommerce RELEASE_VERSION: RC, final and dot-releases progress tracking"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue provides visibility on the progress of the release process of WooCommerce core RELEASE_VERSION and 
+        centralizes any conversations about it. The ultimate goal of this issue is to keep a reference of the steps, 
+        resources, work, and conversations about this release so it can be helpful for the next contributors releasing 
+        a new WooCommerce core version.
+
+        -   WooCommerce core version to release: RELEASE_VERSION ([milestone](https://github.com/woocommerce/woocommerce/milestone/MILESTONE_ID))
+        -   Release lead: RELEASE_LEAD
+        -   RC release date: RELEASE_DATE
+        -   Final release date: RELEASE_DATE
+        -   WooCommerce core change log: [here](https://github.com/woocommerce/woocommerce/blob/trunk/changelog.txt)
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Resources
+
+        -   [Release calendar](https://developer.woocommerce.com/release-calendar/)
+        -   [Previous releases](https://developer.woocommerce.com/releases/)
+        -   Latest (version 9.9+) [release process changes](https://developer.woocommerce.com/2025/03/05/upcoming-changes-to-the-woocommerce-release-process/)
+        -   Internal detailed reference to the release process: Pf59Ax-7F-p2
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Checklist
+
+  - type: markdown
+    attributes:
+      value: |
+        ### RC release
+        
+        #### Get release artifact ready
+
+        -    [] Ensure minimally required WordPress and PHP versions are up-to-date
+        -    [] Ensure all [CFEs](https://github.com/woocommerce/woocommerce/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22code%20freeze%20exception%22) for the target release version are addressed and merged
+        -    [] Validate stable tag in `readme.txt` for the release branch and update if necessary
+        -    [] Validate stable tag in `readme.txt` for the SVN `trunk` ([here](https://plugins.svn.wordpress.org/woocommerce/trunk/readme.txt))
+        -    [] Skip changelog generation (?)
+        -    [] Create a PR to bump version in source code with `GITHUB_TOKEN=<your_PAT_token> pnpm utils code-freeze version-bump -b release/9.9 9.9.0-rc.1`
+        -    [] Review (ensure PR CI-checks are passing) and merge the PR (ensure merge CI-checks are passing)
+        -    [] Build zip file ([workflow](https://github.com/woocommerce/woocommerce/actions/workflows/release-build-zip-file.yml))
+        -    [] Validate version and stable tag in 'woocommerce.php' and `readme.txt` for the generated zip file
+
+        #### Release to WordPress.org
+
+        -    [] Skip notifying Atomic
```

Comment:
Is this a task to not do something? 😆

Reply:
Yep, if someone is fresh into the release process, I think this will be helpful =)